src/styled.js

```diff
@@ -27,18 +33,25 @@ const styled = ({tagName, elementStyle, mountSheet}: StyledArgs) => {
   return class StyledElement extends Component {
     static tagName: string = tagName
     static style: ComponentStyleType = elementStyle
+    static contextTypes = {
+      [channel]: PropTypes.object
+    }
```

Comment:
there is a problem with `themeListener.contextTypes`, because it's `required`, so user required to provide that channel in the context. but I don't think that `styled-jss` should be used only with `ThemeProvider`